app/controllers/voice_controller.rb

```diff
@@ -0,0 +1,39 @@
+class VoiceController < ApplicationController
+  REPEAT_COUNT = 5 # number of times we allow users to repeat their code
```

Comment:
since this is a constant, what about naming it something clearer rather than requiring comment? eg: `NUMBER_OF_TIMES_USER_CAN_REPEAT_CODE`